src/Native/WebGL.js

```diff
@@ -533,7 +549,8 @@ var _elm_community$elm_webgl$Native_WebGL = function () {
 
     LOG('Render canvas');
     var canvas = document.createElement('canvas');
-    var gl = canvas.getContext('webgl') || canvas.getContext('experimental-webgl');
+    var attributes = {stencil: true}
```

Comment:
I don't think this should be a default setting, many games don't need the stencil buffer.

Better would be to handle this as mentioned here: https://github.com/elm-community/elm-webgl/issues/19